### PR TITLE
Document breaking change in HTTPS Redirection Middleware

### PR DESCRIPTION
**Pull request by scottaddie**
Document the breaking change described at https://github.com/aspnet/Announcements/issues/448.

### CODE REVIEW

File: docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+<!--
+
+## Category
+
+ASP.NET Core
+
+## Affected APIs
+
+[UseHttpsRedirection](/dotnet/api/microsoft.aspnetcore.builder.httpspolicybuilderextensions.usehttpsredirection?view=aspnetcore-5.0)
```
Comment (gewarren) [inline docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md:63]:
```suggestion
`Overload:Microsoft.AspNetCore.Builder.HttpsPolicyBuilderExtensions.UseHttpsRedirection`
```

File: docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+author: scottaddie
+ms.author: scaddie
+ms.date: 02/04/2021
+---
+# Middleware: HTTPS Redirection Middleware throws exception on ambiguous HTTPS ports
+
+In ASP.NET Core 6.0, the [HTTPS Redirection Middleware](xref:Microsoft.AspNetCore.Builder.HttpsPolicyBuilderExtensions.UseHttpsRedirection%2A) throws an exception of type <xref:System.InvalidOperationException> when it finds multiple HTTPS ports in the server configuration. The exception's message contains the following text:
+
+```console
+"Cannot determine the https port from IServerAddressesFeature, multiple values were found. Set the desired port explicitly on HttpsRedirectionOptions.HttpsPort."
```
Comment (gewarren) [inline docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md:13]: I don't normally code fence error messages, but if this is standard for your docs, it's fine.

File: docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+If there are multiple distinct ports, it's unclear which port should be used. The middleware throws an exception and fails the HTTP request.
+
+## Reason for change
+
+This change prevents potentially sensitive data from being served over unencrypted HTTP connections when HTTPS is known to be available.
+
+## Recommended action
+
+To enable HTTPS redirection when the server has multiple distinct HTTPS ports, you must specify one port in the configuration. For more information, see [Port configuration](/aspnet/core/security/enforcing-ssl?view=aspnetcore-5.0#port-configuration).
```
Comment (gewarren) [inline docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md:45]: Do you need to add `preserve_view=true` in the link?

**Review by gewarren — DISMISSED** [review]

File: docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Breaking change: Middleware: HTTPS Redirection Middleware throws exception on ambiguous HTTPS ports"
+description: "Learn about the breaking change in ASP.NET Core 6.0 titled Middleware: HTTPS Redirection Middleware throws exception on ambiguous HTTPS ports"
+author: scottaddie
+ms.author: scaddie
+ms.date: 02/04/2021
+---
+# Middleware: HTTPS Redirection Middleware throws exception on ambiguous HTTPS ports
+
+In ASP.NET Core 6.0, the [HTTPS Redirection Middleware](xref:Microsoft.AspNetCore.Builder.HttpsPolicyBuilderExtensions.UseHttpsRedirection%2A) throws an exception of type <xref:System.InvalidOperationException> when it finds multiple HTTPS ports in the server configuration. The exception's message contains the following text:
```
Comment (gewarren) [inline docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md:10]:
```suggestion
In ASP.NET Core 6.0, [HTTPS Redirection Middleware](xref:Microsoft.AspNetCore.Builder.HttpsPolicyBuilderExtensions.UseHttpsRedirection%2A) throws an exception of type <xref:System.InvalidOperationException> when it finds multiple HTTPS ports in the server configuration. The exception's message contains the following text:
```

File: docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md (hunk 1 above)
Comment (scottaddie) [reply docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md]: Thank you for catching this!

**Comment by scottaddie** [comment]
@gewarren Ready for your review

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

**Pull request merged**